src/components/POLine/Item/ProductIdDetailsForm.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { useCallback } from 'react';
```

Comment:
basically, this is a revert of previous recent "fix", since it just hides the issue deeper and breaks the functionality of restoring link in case user adds and removes productId